vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -11,7 +11,7 @@ def textual_group_properties
   end
 
   def textual_group_relationships
-    items = %w(ems container_groups lives_on)
+    items = %w(ems container_groups container_services container_replicators containers lives_on)
```

Comment:
Order should be: `ems container_services container_replicators container_groups containers lives_on`